site/quickstart.md

```diff
@@ -98,12 +98,12 @@ $ kubectl -n traefik get pod -w
 
 a.	Create a new image with a domain home by running the [`create-domain`](../kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script. Follow the directions in the [README](../kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md) file, including:
 
-* Modifying the sample `inputs.yaml` file with the `domainUID` (`sample-domain1`) and domain namespace (`sample-domains-ns1`).
+* Modifying the sample `inputs.yaml` file with the `domainUID` (`sample-domain1`), domain namespace (`sample-domains-ns1`) and the base image (`oracle/weblogic:12213-patch-wls-for-k8s`).
 
-* Creating Kubernetes secrets `username` and `password` of the admin account using the [`create-weblogic-credentials`](../kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh) script.
+* Creating Kubernetes secrets `username` and `password` using the [`create-weblogic-credentials`](../kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh) script:
 ```
 cd ../kubernetes/samples/scripts/create-weblogic-domain-credentials
-./create-weblogic-credentials.sh -u <username> -p <password> -n sample-domain1-ns -d sample-domain1
+./create-weblogic-credentials.sh -u <weblogic> -p <welcome1> -n sample-domain1-ns -d sample-domain1
 ```
 
 b.	Confirm that the operator started the servers for the domain:
```

Comment:
when the user does kubectl get po -n sample-domain1-ns please specify that the user will see the admin server and managed servers running.

Reply:
Got it.